frontend/src/app/util/code_gen/style_css_formats.cljs

```diff
@@ -53,28 +53,30 @@
     :else value))
 
 (defn format-color
-  [value _options]
-  (cond
-    (:image value)
-    (let [image-url (cfg/resolve-file-media (:image value))
-          opacity-color (when (not= (:opacity value) 1)
-                          (uc/gradient->css {:type :linear
-                                             :stops [{:color "#FFFFFF" :opacity (:opacity value)}
-                                                     {:color "#FFFFFF" :opacity (:opacity value)}]}))]
-      (if opacity-color
-        ;; CSS doesn't allow setting directly opacity to background image, we should add a dummy gradient to get it
-        (dm/fmt "%, url(%) no-repeat center center / cover" opacity-color image-url)
-        (dm/fmt "url(%) no-repeat center center / cover" image-url)))
-
-    (not= (:opacity value) 1)
-    (uc/color->background value)
-
-    :else
-    (str/upper (:color value))))
+  [value options]
+  (let [format (or (:format options) :hex)]
```

Comment:
in this case i think it can be
```
(get options :format :hex)
```

Reply:
Awesome!! I'll update it, thanks for the tip